src/eko/io/metadata.py

```diff
@@ -55,13 +57,21 @@ def load(cls, path: os.PathLike):
 
         Returns
         -------
-        bool
+        Metadata
             loaded metadata
         """
         path = pathlib.Path(path)
-        content = cls.from_dict(
-            yaml.safe_load(InternalPaths(path).metadata.read_text(encoding="utf-8"))
-        )
+        paths = InternalPaths(path)
+        # read raw file first to catch version
+        raw = yaml.safe_load(paths.metadata.read_text(encoding="utf-8"))
+        version = parse(raw["version"])
+        # patch if necessary
+        if version.major == 0 and version.minor == 13:
```

Comment:
I wonder if we should add a case also for the `0.0.0` ?

Reply:
In the end we decided that no